E2-Breakdown-Money/src/main/java/com/redelles/e2/Numbers.java

```diff
@@ -2,73 +2,75 @@
 
 import java.util.Scanner;
 
-public class Utils {
+public class Numbers {
 
-    public static Integer getAmount(){
+    public static Integer scanNumber(){
 
-        Integer amount = -1;
-        Boolean flag;
+        Integer numberInput = -1;
 
-        do{
+        do {
             try {
-                System.out.println("Please enter an amount of money.");
-                amount = Utils.scanNumber();
-                flag = false;
+
+                Scanner scan = new Scanner(System.in);
+                numberInput = scan.nextInt();
+
             } catch (Exception e) {
                 System.out.println("REMEMBER This application only works with numbers.");
-                flag = true;
             }
+        }while( numberInput == -1);
 
-        } while (flag);
-
-        return amount;
+        return numberInput;
     }
 
-    public static Integer scanNumber(){
+    public static Integer getAmount(){
 
-        Integer numberInput = 0;
-        Scanner scan = new Scanner(System.in);
-        numberInput = scan.nextInt();
+        Integer amount = -1;
 
-        return numberInput;
-    }
+        do{
+            try {
+                System.out.println("Please enter an amount of money.");
+                amount = Numbers.scanNumber();
+
+            } catch (Exception e) {
+                System.out.println("REMEMBER This application only works with numbers.");
+            }
 
-    public static Integer breakdown(Integer quantity, Integer result){
+        } while (amount == -1);
 
-        Integer breakdownQty = result / quantity;
-        System.out.println(breakdownQty + " -> " + quantity);
-        return result = result - quantity * breakdownQty;
+        return amount;
     }
 
     public static Integer getQty(){
 
         Integer qty = -1;
-        Boolean flag;
 
         do{
             try {
                 System.out.println("Please enter a quantity.");
-                qty = Utils.scanNumber();
-                flag = false;
+                qty = Numbers.scanNumber();
+
             } catch (Exception e) {
                 System.out.println("REMEMBER This application only works with numbers.");
-                flag = true;
+
             }
 
-        } while (flag);
+        } while (qty == -1);
 
         return qty;
     }
 
     public static Integer[] getArrayOfThree(){
```

Comment:
Why are you only taking two numbers for the breakdown?? You should able to take as much as the user wants.